Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -1057,6 +1057,19 @@
     (allow mach-kernel-endpoint
         (apply-message-filter
             (deny mach-message-send (with telemetry))
+#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000
```

Comment:
This would be better as a `HAVE` macro, rather than embed these specific version numbers in the file.

Reply:
That is a good point. Will fix!